workers-docs/src/content/tooling/wrangler/commands.md

```diff
@@ -12,6 +12,7 @@ weight: 2
   * [Registered domain](#publishing-to-your-own-domain)
   * [Multiple domains](#publishing-the-same-code-to-multiple-places)
 - [👂 `dev`](#dev-alpha-))
+- [🦚 `tail`](#tail-))
```

Comment:
```suggestion
- [🦚 `tail`](#tail-)
```

Reply:
DId you test this link works locally? @ashleymichal

Reply:
pushed fix!